### ISSUE TITLE
Floating Toolbar/DockPanels with Resizable & Win 10

### ISSUE THREAD

**Issue by hpw4711** [plashenkov/TBX#12]
As it goes for an 13 th... 

If you use a floating & resizable toolbar and/or dockpanel on Win 10 (resizable = false is OK!!) the NC caption get's ugly with a WHITE bar at the top of the Toolwindow.

![image](https://user-images.githubusercontent.com/11184677/39969037-2bacb8b0-56ce-11e8-978a-bfe3337e98f0.png)

Currently I am working to fix it, but looks ugly while on

1. All Themes: Border.Y := GetSystemMetrics(YMetrics[Resizable]) is not used to a common function
  
2. also the NC caption drawing requires adapation

while the ugly part on Win 10 and new Windows-Themes, the border width is now 1 pixel and may also Windows- Themes dependent.

**Comment by hpw4711** [comment]
Well my solution so far to get the frames right:

![image](https://user-images.githubusercontent.com/11184677/40049579-0fc206a4-582d-11e8-92b5-1ffd800a9dbb.png)

the change was done at TB2Dock unit:

procedure TTBFloatingWindowParent.CreateParams(var Params: TCreateParams);

snip change:

    if not(csDesigning in ComponentState) then begin
      // gets ugly white top NC frame
      //  Style := WS_POPUP or WS_BORDER or ThickFrames[FDockableWindow.FResizable];

      // hpw1398
      Style := WS_POPUP or WS_BORDER;
    end

**Comment by plashenkov** [comment]
Yes, it looks like there is no better way to do this. But let's remove WS_THICKFRAME for Windows 10 only. Which is the best method to detect we are on Windows 10?

**Comment by plashenkov** [comment]
For reference: https://stackoverflow.com/questions/39731497/create-window-without-titlebar-with-resizable-border-and-without-bogus-6px-whit
